app/controllers/events_controller.rb

```diff
@@ -1,6 +1,11 @@
 class EventsController < ApplicationController
   def index
     @events = meetup.self_events
+    @events = @events.select do |event|
```

Comment:
I think it's better to move this logic to the service, instead of adding it to this controller.

What do you think about creating a separated method that would check if the user is included in the `event_hosts` array?